speechmatics/cli.py

```diff
@@ -576,7 +575,6 @@ def parse_args(args=None):
     )
     transcribe_subparser.add_argument(
         "--max-delay-mode",
-        default="flexible",
         choices=["fixed", "flexible"],
```

Comment:
I think the choices should be swapped round so that the (current) server-side default, "flexible", is first in the list. It's a semi-convention that defaults are first.